Conceptpower+Spring/src/test/java/edu/asu/conceptpower/servlet/rest/ConceptIDLookupTest.java

```diff
@@ -0,0 +1,94 @@
+package edu.asu.conceptpower.servlet.rest;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.conceptpower.core.ConceptEntry;
+import edu.asu.conceptpower.root.TypeDatabaseClient;
+import edu.asu.conceptpower.root.URIHelper;
+import edu.asu.conceptpower.servlet.core.ConceptTypesService;
+import edu.asu.conceptpower.servlet.core.ConceptTypesService.ConceptTypes;
+import edu.asu.conceptpower.servlet.core.IConceptManager;
+import edu.asu.conceptpower.servlet.xml.XMLConceptMessage;
+import edu.asu.conceptpower.servlet.xml.XMLMessageFactory;
+import junit.framework.Assert;
+
+public class ConceptIDLookupTest {
+
+    @Mock
+    private IConceptManager dictManager;
+
+    @Mock
+    private TypeDatabaseClient typeManager;
+
+    @Mock
+    private XMLMessageFactory messageFactory;
+
+    @Mock
+    private IConceptManager conceptManager;
+
+    @Mock
+    private ConceptTypesService conceptTypesService;
+
+    @Mock
+    private URIHelper uriCreator;
+
+    @InjectMocks
+    private ConceptIDLookup conceptIDLookup;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+
+        Mockito.when(messageFactory.createXMLConceptMessage()).thenReturn(new XMLConceptMessage(uriCreator));
+    }
+
+    @Test
+    public void testGetConceptByIdForAlternativeIDs() throws Exception {
+        ConceptEntry entry = new ConceptEntry();
+        entry.setId("CONf375adff-dde7-4536-9e62-f80328f800d0");
+        entry.setWordnetId("W-ID1, W-ID2");
+        Mockito.when(dictManager.getConceptEntry("CONf375adff-dde7-4536-9e62-f80328f800d0")).thenReturn(entry);
+        String s = conceptIDLookup
+                .getConceptById("http://www.digitalhps.org/concepts/CONf375adff-dde7-4536-9e62-f80328f800d0")
+                .toString();
```

Comment:
why do you make that a string and not testing if the returned response body contains what it should and has the correct status?